### PR TITLE
nf-test + fix 3 modules

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/manta/convertinversion/environment.yml [hunk 1]
```diff
@@ -1,9 +1,11 @@
 name: manta_convertinversion
+
```
Comment (maxulysse) [inline modules/nf-core/manta/convertinversion/environment.yml:2]: why the empty lines?

**Review by maxulysse — APPROVED** [review]

File: modules/nf-core/manta/convertinversion/environment.yml (hunk 1 above)
Comment (nvnieuwk) [reply modules/nf-core/manta/convertinversion/environment.yml:2]: That's `prettier`'s fault :grin:

Comment (maxulysse) [reply modules/nf-core/manta/convertinversion/environment.yml:2]: it hasn't happened in your other modules `environment.yml` file.
check modules/nf-core/delly/call/environment.yml
I'd rather we are consistent and not have useless empty lines

Comment (nvnieuwk) [reply modules/nf-core/manta/convertinversion/environment.yml:2]: Huh that's weird... I'll remove them here then

**Review by maxulysse — APPROVED** [review]

**Pull request merged**